src/main/java/io/cdap/plugin/sfmc/sink/RecordDataExtensionRowConverter.java

```diff
@@ -93,7 +93,7 @@ private String getLogicalType(Schema.LogicalType logicalType, String fieldName,
     }
   }
 
-  private String getType(Schema.Type type, String fieldName, Object val) {
+  public String getType(Schema.Type type, String fieldName, Object val) {
```

Comment:
is this used in tests in a different package? If not, just make it package private. Similar comment for other methods made public

Reply:
Done